docs/pages/repo/docs/getting-started/create-new.mdx

```diff
@@ -140,70 +140,84 @@ You'll see that `"web"` depends on a package called `"ui"`.
     ```json filename="apps/web/package.json"
     {
       "dependencies": {
-        "ui": "workspace:*"
+        "@repo/ui": "workspace:*"
       }
     }
     ```
   </Tab>
 </Tabs>
 
-This means that our **web app depends on our local `ui` package**.
+This means that our **web app depends on our local `@repo/ui` package**.
 
-If you look inside `apps/docs/package.json`, you'll see the same thing. Both `web` and `docs` depend on `ui` - a shared component library.
+If you look inside `apps/docs/package.json`, you'll see the same thing. Both `web` and `docs` depend on `@repo/ui` - a shared component library.
 
 This pattern of sharing code across applications is extremely common in monorepos - and means that multiple apps can share a single design system.
 
 #### Understanding imports and exports
 
-Take a look inside `./apps/docs/app/page.tsx`. Both `docs` and `web` are [Next.js](https://nextjs.org/) applications, and they both use the `ui` library in a similar way:
+Take a look inside `./apps/docs/app/page.tsx`. Both `docs` and `web` are [Next.js](https://nextjs.org/) applications, and they both use the `@repo/ui` library in a similar way:
 
 ```tsx filename="apps/docs/app/page.tsx"
-import { Button, Header } from "ui";
-//       ^^^^^^^^^^^^^^         ^^
+import { Button } from "@repo/ui/button";
+//       ^^^^^^         ^^^^^^^^^^^^^^^
 
 export default function Page() {
   return (
     <>
-      <Header text="Docs">
-      <Button />
+      <Button appName="web" className={styles.button}>
+        Click me!
+      </Button>
     <>
   );
 }
 ```
 
-They're importing `Button` directly from a dependency called `ui`! How does that work? Where is `Button` coming from?
+They're importing `Button` directly from a dependency called `@repo/ui/button`! How does that work? Where is `Button` coming from?
 
 Open `packages/ui/package.json`. You'll notice these two attributes:
```

Comment:
"these two attributes" doesn't make sense anymore since it's now the `exports` field. Could you reword this to "You'll notice the `exports` field:"

Reply:
That is much better, thank you! 🤩